### ISSUE TITLE
Feature request: SGD, adaptive learning rate, early stopping

### ISSUE THREAD

**Issue by jaagut** [eriklindernoren/PyTorch-YOLOv3#616]
Ideas for additional features:

- use sgd with burn in and parameters from config file
- adaptive learning rate
- early stopping (in argparse)
